PatreonClient/Models/BaseRelationship.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using PatreonClient.Responses;
+
+namespace PatreonClient.Models
+{
+    public abstract class BaseRelationShip : IRelationship
```

Comment:
I am being very picky here, but can we lower case that `S` please 😄

Reply:
No worries, not sure how that slipped through ... 🤣